renovate.json

```diff
@@ -9,7 +9,21 @@
 		{
 			"packagePatterns": [ "^@wordpress" ],
 			"groupName": "wordpress-monorepo",
-			"separateMajorMinor": false
+			"separateMajorMinor": false,
+			"prPriority": 2
+		},
+		{
+			"packagePatterns": [
+				"^@babel",
+				"^babel-",
+				"^lodash$",
+				"^react-redux$",
+				"^react$",
```

Comment:
Also `^react-dom$`

Reply:
Asked @rarkins over here: https://github.com/renovatebot/config-help/issues/414

Reply:
reworked some of these to use the existing monorepo defs